src/standard/BaseStandardVerifier.sol

```diff
@@ -1,11 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2022 Aztec
 pragma solidity >=0.8.4;
-// gas cost at 5000 optimizer runs 0.8.10: 287,589 (includes 21,000 base cost)
```

Comment:
As we discussed, I was not isolating the contract costs sufficiently, but you guided me to use the `--gas-report` flag and We calculated a gas estimate of 261808. This is not apples-to-apples because the former estimate was calculated using different tooling. Could you update the comment to reflect the new estimate?